doc/decisions/DR014-Management-policy-responses.md

```diff
@@ -0,0 +1,256 @@
+  at a later date (e.g. the color space for the `colorManaged` trait, if
+  the manager can't ever provide this).
+
+### Option 2 - Exact Matching
+
+It should respond only to sets where it is capable of resolving or
+persisting the required data:
+
+```python
+if traitSet == { FileTrait.kId }:
+    ManagedTrait.imbueTo(policy)
+```
+
+Such that it _only_ opts into managing the specifically limited trait
+set that it can support:
+
+```python
+policies = [
+    {},  # Main document
+    {}   # Generated Images
+]
+```
+
+The Host would receive empty responses (`{}`) for both the original
+trait sets, and so must potentially degrade and re-try its queried trait
+set if it is happy with only a subset. e.g:
+
+```python
+traitSets = [
+    { 'file', 'nodeGraph' },                        # Main document
+    { 'file', 'image', 'raster', 'colorManaged' },  # Generated images
+    { 'file', 'image', 'raster' },
+    { 'file', 'image' },
+    { 'file' }                                      # File
+]
+```
+
+Resulting in:
+
+```python
+policies = [
+    {},                    # Main document
+    {},                    # Generated Images
+    {},
+    {},
+    { 'managed': {...} },  # File
+]
+```
+
+#### Pros
+
+- Conveys only what traits are supported to the host, allowing
+  behaviour to be properly adapted to the manager's capabilities.
+
+#### Cons
+
+- Host implementation is significantly more complicated.
+- No way to hint at any additional traits that may be resolvable
+  outside the typing trait set.
+
+### Option 3 - Supported Traits Response
+
+The host should also include any additional traits that it may attempt
+to resolve in the future, e.g. if it supported the ability to customise
+t,he file format used for writing images, by resolving a custom trait:
+
+```python
+traitSets =[
+    { 'file', 'nodeGraph' },
+    { 'file', 'image', 'raster', 'colorManaged', 'fileFormatOptions' }
+]
+```
+
+The manager should respond as per Option 1, but in addition, imbue any
+of the queried traits that the manager is capable of resolving:
+
+```python
+if FileTrait.kId in traitSet:
+    ManagerTrait.imbueTo(policy)
+    FileTrait.imbueTo(policy)
+```
+
+This would then look like the following:
+
+```python
+policies = [
+    { 'managed': {...}, 'file': {} },  # Main document
+    { 'managed': {...}, 'file': {} }   # Generated Images
+]
+```
+
+This indicates that the manager would like to handle interactions for
+entities with these traits sets, but it can only resolve the `file`
+trait.
+
+> Note:
+> We keep the concept additionally imbuing the `managed` trait
+> (though it may seem superfluous, as an empty trait set is equivalent to
+> un-managed) as in practical use cases it may well have other properties
+> that cover topics such as exclusivity/etc.
+> There are also additional traits that may be imbued by the manager to
+> determine how it handles various aspects of the publishing process,
+> e.g. thumbnails.
```

Comment:
[Question] It's not clear to me why these other properties wouldn't become traits of their own (ie, `ExclusiveContentTrait`,) rather than being ties to the `Managed` trait.

Reply:
In some cases they may, but the "exclusive" flag for the managed trait felt like it made more sense to be part of it, as it is increased detail about what kind of "managed". We generally have "more detail about a specific behaviour" as properties and "different behaviours" as traits.